test/embedding/embedding_test.dart

```diff
@@ -191,77 +189,6 @@ main() {
     );
   });
 
-  test('analytics', () async {
```

Comment:
Is there no way with unified_analytics that we can test that the events are as expected?

Reply:
Ah - only read your explanation now - I guess testing via dartdev is good enough

Reply:
Yeah, I know it's not ideal, but I figure it's good enough for now. If we feel we still want the test coverage in this repo, we can always add it in later when we have the cycles.